slemicro/README.md

```diff
@@ -183,6 +183,14 @@ worker-1   Ready    <none>                      15s   v1.25.9+k3s1
 
 **TIP:** To run the VM creation in parallel you can use `for file in vm-*; do ./create_vm.sh -f ${file} &; done; wait` (the output will be a little bit messy however)
 
+### Static IPs
+
+It is possible to deploy a VM with a static IP by setting the `VM_STATIC_IP` variable.
+
+Optionally `VM_GATEWAY_IP` may also be specified (defaults to `192.168.122.1`).
+
+Note that in this configuration you must first disable DHCP for your libvirt network, which can be achieved via `virsh net-edit` to remove the `<dhcp>` stanza, then `virsh net-destroy` followed by `virsh net-start`
```

Comment:
Why is this needed? Can you just modify the libvirt network to shrink the dhcp pool?

By default it looks like:
```
  <ip address='192.168.122.1' netmask='255.255.255.0'>
    <dhcp>
      <range start='192.168.122.10' end='192.168.122.254'/>
    </dhcp>
  </ip>
```

Reply:
I tried that and shrinking the DHCP pool isn't sufficient - because the then VM gets an IP via DHCP prior to the static configuration.

I could probably find a workaround for that, but the main aim here is to enable testing of DHCP-less scenarios where only static configuration is possible.